### ISSUE TITLE
Invalid object name when deleting with ModelBuilder.HasDefaultSchema set

### ISSUE THREAD

**Issue by GiuseppePiscopo** [PomeloFoundation/Lolita#14]
Not sure if this is a bug, or just some information I'm missing. I've searched through readme and a bit of code and I could not find anything related.

### Description

In my custom `DbContext`, when overriding `OnModelCreating()`, I'm setting a custom default schema with `modelBuilder.HasDefaultSchema("whatever")`.

When I perform a classic, non-batched deletion EF Core generates SQL code similar to:

```sql
SELECT [login].[IpAddress] /* ... other fields */
FROM [whatever].[FailedLogins] AS [login]
WHERE [login].[ModifiedAt] < @__retentionDate_0

DELETE FROM [whatever].[FailedLogins]
WHERE [IpAddress] = @p0;
```

When instead I try with Lolita batch delete, generated SQL code looks like:

```sql
DELETE FROM [FailedLogins]
WHERE [FailedLogins].[ModifiedAt] < '2017-10-18T22:38:11.701Z';
```

which misses the schema in `FROM ...` and hence causes an error:

```
System.Data.SqlClient.SqlException: Invalid object name 'FailedLogins'.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
[...omitted...]
```

### Environment

* Pomelo.EntityFrameworkCore.Lolita 1.1.0
* Microsoft.EntityFrameworkCore 1.1.2
* Microsoft.NETCore.App 1.1.4

**Comment by GiuseppePiscopo** [comment]
Additional test cases:

**modelBuilder.Entity<>.ToTable()**

Tried explicitly setting schema on table during `OnModelCreating`:

```c#
var entityBuilder = modelBuilder.Entity<FailedLoginData>();

entityBuilder.ToTable("FailedLogins", "whatever");
```

but generated SQL stays the same:

```sql
DELETE FROM [FailedLogins]
WHERE [FailedLogins].[ModifiedAt] < '2017-10-18T23:09:13.750Z';
```

**Table() attribute**

Tried explicitly setting schema on table through `[Table()]` attribute:

```c#
[Table("FailedLogins", Schema = "whatever")]
public class FailedLoginData
{
  // ... fields
}
```

and generated SQL now correctly takes schema into account:

```sql
DELETE FROM [whatever].[FailedLogins]
WHERE [whatever].[FailedLogins].[ModifiedAt] < '2017-10-18T23:16:00.777Z';
```

Now, this workaround is not feasible in our scenario as schema is not a string literal, but comes instead from appsettings.

HTH

**Comment by GiuseppePiscopo** [comment]
This should be closed by PR #15
